akka-docs/src/main/paradox/discovery/index.md

```diff
@@ -0,0 +1,245 @@
+# Discovery
+
+@@@ warning
+
+This module is currently marked as @ref:[may change](../common/may-change.md)
+This means that API or semantics can change without warning or deprecation period.
+
+@@@
+
+Akka Discovery provides an interface around various ways of locating services. The built in methods are:
+
+* Configuration
+* DNS
+* Aggregate
+
+In addition [Akka Management](https://developer.lightbend.com/docs/akka-management/current/) contains methods for:
```

Comment:
Perhaps 'such as', so we can add more mechanisms there without having to update the list here?